Insertion_Sort/Insertion_Sort.rs

```diff
@@ -0,0 +1,43 @@
+use std::io;
+use std::str::FromStr;
+
+fn main() {
+    println!("Enter the numbers to be sorted (separated by space) : ");
+    let mut i = read_values::<i32>().unwrap();
+    insertion_sort(&mut i);
+    println!("Sorted array: {:?}", i)
+    }
```

Comment:
This bracket does not need indentation. Please shift this to fall in line with f of fn main()